### PR TITLE
Add pricing info everywhere

### PR DESCRIPTION
**Pull request by shainaraskas**
To resolve confusion about the ES serverless pricing model, I'm adding more explicit detail about how the minimum # of running VCUs impact billing, and what those minimum VCUs are used for.

The following pages got additional info, largely in the form of caution blocks:

[Elasticsearch billing dimensions](https://elastic-dot-co-docs-production-dng2tyfar-elastic-dev.vercel.app/current/serverless/elasticsearch/elasticsearch-billing) - big note re: min VCUs, with detail explaining what the VCUs are for. [links here](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless)

could optionally add a note about free trials here.

image

[Elasticsearch get started](https://elastic-dot-co-docs-production-dng2tyfar-elastic-dev.vercel.app/current/serverless/elasticsearch/get-started) - same big note re: min VCUs.  [links here](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless)

could optionally add a note about free trials here.

image

[Monitor usage](https://elastic-dot-co-docs-production-dng2tyfar-elastic-dev.vercel.app/current/serverless/general/monitor-usage) - hinted at ongoing minimum usage

image

[Manage billing of your organization](https://elastic-dot-co-docs-production-dng2tyfar-elastic-dev.vercel.app/current/serverless/general/manage-billing) - added link [further into the billing dimensions content](https://elastic-dot-co-docs-production-dng2tyfar-elastic-dev.vercel.app/current/serverless/general/serverless-billing)

image

### CODE REVIEW

**Comment by elasticdocs** [comment]
:rocket: Built _elastic-dot-co-docs-preview-docs_ successfully!

* https://elastic-dot-co-docs-production-4yz799tsk-elastic-dev.vercel.app

* Built with commit 7af023908e58fcffef89f8160b68e8f3f0a6a65e

> Issues? Visit #next-docs in Slack

File: serverless/pages/sign-up.mdx [hunk 1]
```diff
@@ -60,7 +60,11 @@ To remove limitations, subscribe to [Elastic Cloud](https://www.elastic.co/guide
...
+You can subscribe to Elastic Cloud at any time during your trial. [Billing](/serverless/general/serverless-billing) starts when you subscribe. To maximize the benefits of your trial, subscribe at the end of the free period. To monitor charges, anticipate future costs, and adjust your usage, check your [account usage](https://www.elastic.co/guide/en/cloud/current/ec-account-usage.html) and [billing history](https://www.elastic.co/guide/en/cloud/current/ec-billing-history.html). 
+
+<DocCallOut color="warning" title="Elasticsearch minimum runtime VCUs">
+	The way that you are billed depends on the project type that you select. Some project types have a minimum number of VCUs that are always allocated and billed to your project. <DocLink slug="/serverless/general/serverless-billing" text="Learn more"/>.
```
Comment (leemthompo) [inline serverless/pages/sign-up.mdx:66]:
```suggestion
	Your billing depends on the project type that you select. Some project types have a minimum number of VCUs that are always allocated and billed to your project. <DocLink slug="/serverless/general/serverless-billing" text="Learn more"/>.
```

File: serverless/partials/minimum-vcus-detail.mdx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<DocCallOut color="warning" title="Minimum runtime VCUs">
+    After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number VCUs are always allocated to your project. These VCUs are used for the following purposes:
```
Comment (leemthompo) [inline serverless/partials/minimum-vcus-detail.mdx:2]:
```suggestion
    When you create an Elasticsearch Serverless project, a minimum number of VCUs are always allocated to your project to maintain basic capabilities. These VCUs are used for the following purposes: 
```

File: serverless/partials/minimum-vcus.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<DocCallOut color="warning" title="Elasticsearch minimum runtime VCUs">
+    When you create an Elasticsearch Serverless project, to maintain basic Elasticsearch ingest and search capabilities, a minimum number VCUs are always allocated and billed to your project.    Learn more about [minimum VCUs on Elasticsearch Serverless](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless).
```
Comment (leemthompo) [inline serverless/partials/minimum-vcus.mdx:2]:
```suggestion
    When you create an Elasticsearch Serverless project, to maintain basic Elasticsearch ingest and search capabilities, a minimum number of VCUs are always allocated and billed to your project. Learn more about [minimum VCUs on Elasticsearch Serverless](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless).
```

**Review by leemthompo — COMMENTED** [review]
Some _nitty_ wording suggestions. I think mentioning free trial option as you suggested makes sense too. :)

File: serverless/partials/minimum-vcus.mdx (hunk 3 above)
Comment (leemthompo) [reply serverless/partials/minimum-vcus.mdx:2]: Consider aligning with previous suggestion and/or breaking long sentence in two

File: serverless/pages/sign-up.mdx (hunk 1 above)
Comment (florent-leborgne) [inline serverless/pages/sign-up.mdx:66]:
```suggestion
	The way that you are billed depends on the project type that you select. Some project types have a minimum number of VCUs that are always allocated to your project, incurring a minimum cost even with no active usage. <DocLink slug="/serverless/general/serverless-billing" text="Learn more"/>.
```
I was just pinged about the UI changes regarding this issue so this is timely. I think one of the pain points from customers is that we're not transparent with the costs, so let's be super explicit in addition to serving a bit of VCU jargon :D

File: serverless/partials/minimum-vcus-detail.mdx (hunk 2 above)
Comment (florent-leborgne) [inline serverless/partials/minimum-vcus-detail.mdx:2]:
```suggestion
    After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number of VCUs are always allocated to your project. These VCUs are used for the following purposes: 
```

File: serverless/partials/minimum-vcus-detail.mdx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<DocCallOut color="warning" title="Minimum runtime VCUs">
+    After you create an Elasticsearch Serverless project, to maintain basic Elasticsearch capabilities, a minimum number VCUs are always allocated to your project. These VCUs are used for the following purposes: 
+
+    - **Ingest**: Ensure constant availability for ingesting data into your project.
+    - **Search**: Maintain a data cache and support low latency searches.
+
+    These minimum VCUs are billed at the standard rate per VCU hour.
```
Comment (florent-leborgne) [inline serverless/partials/minimum-vcus-detail.mdx:7]:
```suggestion
    These minimum VCUs are billed at the standard rate per VCU hour, incurring a minimum cost even when you're not actively using your project.
```

File: serverless/partials/minimum-vcus.mdx (hunk 3 above)
Comment (florent-leborgne) [inline serverless/partials/minimum-vcus.mdx:2]:
```suggestion
    When you create an Elasticsearch Serverless project, to maintain basic Elasticsearch ingest and search capabilities, a minimum number of VCUs are always allocated and billed to your project, incurring a minimum cost even with no active usage. Learn more about [minimum VCUs on Elasticsearch Serverless](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless).
```

**Review by florent-leborgne — CHANGES_REQUESTED** [review]
More transparency, nice!! 

2 things on my side regarding these changes:
- Let's be explicit that those ever running VCUs mean minimum cost for the user even when they're not actively using their project.
- The more the merrier but I think it would be sufficient to call this out in the Elasticsearch billing dimensions page, the [project creation page](https://www.elastic.co/docs/current/serverless/elasticsearch/get-started), and the usage monitoring page

File: serverless/pages/sign-up.mdx (hunk 1 above)
Comment (florent-leborgne) [reply serverless/pages/sign-up.mdx:66]: In the product, we're being explicit that the minimum cost is 1.25 ECU / hour. I guess we're not showing numbers in the docs, but maybe we should link more often to the website's pricing page? WDYT?

Comment (shainaraskas) [reply serverless/pages/sign-up.mdx]: I've gone through and applied your suggestions 🎉  - now, every time we talk about VCUs, we're linking out to the pricing page. If you have any suggestions for additional places to link to the pricing page, I'm open to them.

**Review by florent-leborgne — DISMISSED** [review]
Nice! 🚀

File: serverless/pages/manage-billing-monitor-usage.mdx [hunk 5]
```diff
@@ -21,3 +21,6 @@ On the **Usage** page you can:
 The usage breakdown information is an estimate. To get the exact amount you owe for a given month, check your invoices in the <DocLink slug="/serverless/general/billing-history" text="billing history"/>.
 </DocCallOut>
 
+<DocCallOut color="warning" title="Elasticsearch minimum runtime VCUs">
+    When you create an Elasticsearch Serverless project, to maintain basic ingest and search capabilities, a minimum number of VCUs are always allocated and billed to your project. These VCUs incur a minimum cost even with no active usage. Learn more about [minimum VCUs on Elasticsearch Serverless](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless).
```
Comment (shainaraskas) [inline serverless/pages/manage-billing-monitor-usage.mdx:25]:
```suggestion
    When you create an Elasticsearch Serverless project, a minimum number of VCUs are always allocated to your project to maintain basic ingest and search capabilities. These VCUs incur a minimum cost even with no active usage. Learn more about [minimum VCUs on Elasticsearch Serverless](https://www.elastic.co/pricing/serverless-search#what-are-the-minimum-compute-resource-vcus-on-elasticsearch-serverless).
```

**Review by kilfoyle — APPROVED** [review]
LGTM! 🦖

**Pull request merged**